grunz/tests/test_file_utils.py

```diff
@@ -0,0 +1,156 @@
+import unittest
+import tempfile
+import os
+import time
+import hashlib
+from pathlib import Path
+from filecmp import cmp
+
+from grunz.file_utils.file_utils import FileUtils
+
+
+class TestFileUtils(unittest.TestCase):
+
+    def setUp(self):
+        # Creates a temporary directory for test files
```

Comment:
The jury is out on comments but I'd say before including one you should see if the code self documents. In this case `# Creates a temporary directory for test files` might not be nessecary given the clarity of the `tempfile` method call.

Reply:
True, that part probably doesn't need a comment, have removed.